repo/repo.go

```diff
@@ -1,6 +1,17 @@
 package repo
 
-import util "github.com/jbenet/go-ipfs/util"
+import (
+	config "github.com/jbenet/go-ipfs/repo/config"
+	util "github.com/jbenet/go-ipfs/util"
+)
+
+type Interface interface {
```

Comment:
can we have a better name for this? `Repo`?  `Interface` is non-descriptive within the package, and outside the package _relies_ on the user naming the import the same thing you named it, which is not always possible.